resources/lib/pigskin.py

```diff
@@ -228,6 +228,31 @@ def get_weeks_games(self, season, season_type, week):
             raise
 
         return sorted(games, key=lambda x: x['gameDateTimeUtc'])
+        
+    def get_team_games(self, season, team=None):
+        try:
+            url = self.config['modules']['ROUTES_DATA_PROVIDERS']['teams']
+            teams = self.make_request(url, 'get')
+            if team is None:
+                return teams
+            else:
+                #Looking for the Teamname
+                for conference in teams['modules']:
+                    if 'content' in teams['modules'][conference]:
+                        for teamname in teams['modules'][conference]['content']:
+                            if team == teamname['fullName']:
+                                team = teamname['seoname']
+
```

Comment:
your right that should be an exception.
I thought it would be ctached in the gui part but your right.